modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectRelationship/EditRelationship.tsx

```diff
@@ -73,7 +73,10 @@ export default function EditRelationship({
 		parameterRequired,
 	});
 
-	const readOnly = !hasUpdateObjectDefinitionPermission || values.reverse;
+	const readOnly =
+		!hasUpdateObjectDefinitionPermission ||
+		values.reverse ||
+		initialValues.system;
```

Comment:
In this specific case shouldn't we keep using `initialValues` since we never want to update the `system` property from the UI?

Reply:
@G-Sa Yeah, we can keep it like that, in this specific case, because it'll not be updated, so the result will be the same. But keep that in mind for other situations.